samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -936,6 +944,31 @@ To perform the DFU procedure, complete the following steps:
             :start-after: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_start
             :end-before: fota_upgrades_over_ble_nrfcdm_common_dfu_steps_end
 
+   .. group-tab:: nRF54H DKs
+
+      1. Observe that **LED 0** is blinking at a 1 second interval, which indicates that the DFU mode is disabled.
+      #. Press the **Button 3** for 7 seconds or more to enter the DFU mode.
+      #. Observe that **LED 0** is blinking at a 0.25 second interval, which indicates that the DFU mode is enabled.
+      #. Observe that **LED 2** is blinking, which indicates that the Fast Pair advertising is enabled.
+      #. Perform DFU using the `nRF Connect Device Manager`_ mobile app:
+
+         1. Generate the SUIT envelope by building your application with the FOTA support over Bluetooth Low Energy.
+            You can find the generated :file:`root.suit` envelope in the :file:`<build_dir>/DFU` directory.
+            Alternatively, you can use the generated :file:`dfu_suit.zip` package in the :file:`<build_dir>/zephyr` directory.
+         #. Download the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package to your device.
+
+            .. note::
+               `nRF Connect for Desktop`_ does not currently support the FOTA process.
+
+         #. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
+
+            a. Ensure that you can access the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package from your phone or tablet.
+            #. In the mobile app, scan and select the device to update.
+            #. Switch to the :guilabel:`Image` tab.
+            #. In the **Firmware Upgrade** section, tap the :guilabel:`SELECT FILE` button and select the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package.
+            #. Tap the :guilabel:`START` button.
+            #. Wait for the DFU to finish and verify that the application works properly.
```

Comment:
You could also inform that:

```
      .. note::
         Support for SUIT updates is available starting from the following versions of the `nRF Connect Device Manager`_ mobile app:

         * Version ``2.0`` on Android.
         * Version ``1.7`` on iOS.
```

Reply:
btw. Since the instructions here are copy-paste from nRF Desktop documentation, maybe we could move them to common place and use include here (similarly as we did for nRF52) ?

Reply:
I think we asked the Charon team for that

Reply:
@tomchy, where would be good place to put the general testing instructions for SUIT DFU so that we link them in this sample documentation and nRF Desktop documentation? (you can take a look on how it's done for MCUboot solution)

Reply:
This will be done as part of the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/19557

Depending on the order of the merge, I will either align this PR or the linked one.